editor/plugins/path_3d_editor_plugin.cpp

```diff
@@ -555,7 +563,8 @@ EditorPlugin::AfterGUIInput Path3DEditorPlugin::forward_3d_gui_input(Camera3D *p
 				real_t dist_to_p = p_camera->unproject_position(gt.xform(c->get_point_position(i))).distance_to(mbpos);
 				real_t dist_to_p_out = p_camera->unproject_position(gt.xform(c->get_point_position(i) + c->get_point_out(i))).distance_to(mbpos);
 				real_t dist_to_p_in = p_camera->unproject_position(gt.xform(c->get_point_position(i) + c->get_point_in(i))).distance_to(mbpos);
-				real_t dist_to_p_up = p_camera->unproject_position(gt.xform(c->get_point_position(i) + c->get_point_baked_posture(i, true).get_column(1))).distance_to(mbpos);
+				float disk_size = EDITOR_DEF("editors/3d_gizmos/gizmo_settings/path3d_tilt_disk_size", 0.8);
```

Comment:
`EDITOR_DEF` defines a new setting, it shouldn't be called multiple times for the same setting.

It's also wasteful to re-query this setting every time we're drawing or forwarding an input. It would be better to move it to the constructor like `path_color` and `path_tilt_color`.

Reply:
fixed

Reply:
Right, I think that's ok for this kind of settings. You can change the `EDITOR_DEF` to `EDITOR_DEF_RST` (`RST` stands for "restart"), which will let the user know that they have to restart for changes to be applied.

Alternatively, you can listen to `NOTIFICATION_EDITOR_SETTINGS_CHANGED` to re-`EDITOR_GET` the changed values. But I assume these are not values that people will change frequently (though maybe getting immediate feedback is needed to see if the new value is good?).

Reply:
changed to EDITOR_DEF_RST for now.